cmd/minishift/cmd/root.go

```diff
@@ -69,6 +77,14 @@ var RootCmd = &cobra.Command{
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		var err error
+
+		if filehelper.Exists(constants.Minipath) {
+			homeDirExists = true
+		}
+		if filehelper.IsEmpty(constants.Minipath) {
```

Comment:
I think having _IsEmpty_ checking for empty directory as well as empty file content is quite unfortunate. I've not have came across an IO lib which does that. I think your initial IsEmptyDir was much better.